src/agent/rustjail/src/seccomp.rs

```diff
@@ -39,6 +39,20 @@ fn get_rule_conditions(args: &[LinuxSeccompArg]) -> Result<Vec<ScmpArgCompare>>
     Ok(conditions)
 }
 
+pub fn check_unknown_syscalls(scmp: &LinuxSeccomp) -> String {
+    let mut unknown_syscalls = String::new();
+
+    for syscall in &scmp.syscalls {
+        for name in &syscall.names {
+            if get_syscall_from_name(name, None).is_err() {
+                unknown_syscalls += format!("{} ", name).as_str();
```

Comment:
If you want to keep the list as a string you can simplify to something like:

```suggestion
                unknown_syscalls += name.to_string();
```

Reply:
I fixed it. Thank you.